index.html

```diff
@@ -9,6 +9,20 @@
     <title>Aziz Mejri - Portfolio</title>
   </head>
   <body>
+    <div class="menu hidden">
+      <div class="close-btn-wrapper">
+        <button class="close-btn">
+          <img src="./imgs/close-btn.svg" alt="close button" />
+        </button>
+      </div>
+      <div class="links-con">
+        <ul>
+          <li><a href="#portoflio">Portoflio</a></li>
+          <li><a href="#about">About</a></li>

```

Comment:
- [OPTIONAL] Good job adding `href` to your links 🙌. However, the **About** menu item won't navigate to the corresponding section of your page. Please consider checking if the _**id**_ of the `about section` and the one in the `menu item` are the same.